osmt_cli.sh

```diff
@@ -371,7 +371,11 @@ load_static_ci_dataset(){
   echo
   echo_info "Loading static CI dataset into MySQL database ${DB_NAME} at ${DB_HOST}:${DB_PORT}"
   echo
-  local -i db_container_count; db_container_count="$(docker ps -q --filter name=${OSMT_STACK_NAME}_db_1 | wc -l)"
+  local -i db_container_count=0;
+  # some installations of Docker delimit container names with hyphens. Others use underscores.
+  db_container_count+="$(docker ps -q --filter name=${OSMT_STACK_NAME}_db_1 | wc -l)"
+  db_container_count+="$(docker ps -q --filter name=${OSMT_STACK_NAME}-db-1 | wc -l)"
+  echo_debug $db_container_count
   if [[ "${db_container_count}" -ne 1 ]]; then
```

Comment:
- [ ] Can/should we change this to check also whether container count > 1? Just incase there is some weirdness in the user environment where they have both name-db-1 and name_db_1. Possibly:
if(count > 1) {
  "it appears there are parallel containers <name1> <name2>, please remove one before proceeding"
} else if (count != 1) {
  current logic>
}
Alternately hide the db_container_count lines behind an if/else if so that it will be restricted to [0, 1]

Reply:
I chose to address this with messaging that accommodates the edge case you called out. What do you think?

Reply:
@Corpratespaz There's part of your question I didn't address; let me recap that, please.

The containers are namespaced with a "stack name", defaulting to `osmt_dev`, but it's overridable with an env var. The likelihood of multiple containers running as exactly `osmt_dev_db_1` and `osmt_dev-db-1` at the same time is so remote that I don't think we need to worry about that.